doc/source/servers/batch.md

```diff
@@ -0,0 +1,149 @@
+# Batch Processing with Seldon Core
+
+Seldon Core provides a command line component that allows for highly parallelizable batch processing with the horizontally scalable seldon core kubernetes model deployments.
+
+For stream processing with Seldon Core please see [Stream Processing with KNative Eventing](../streaming/knative_eventing.md).
+
```

Comment:
The stream link will be a top level section when kafka PR included. Need to ensure this is changed.

Reply:
Done